.github/actions/repo-gardening/src/index.js

```diff
@@ -22,6 +23,11 @@ const automations = [
 		event: 'push',
 		task: addMilestone,
 	},
+	{
+		event: 'pull_request',
+		action: [ 'opened', 'synchronize', 'edited', 'labeled' ],
```

Comment:
Happy to add reopened, but I think we should leave `unlabeled` out, just in case someone purposefully wants to remove a label; the action would then keep adding it over after your changes. I would assume that someone removing a label that was added by the action does it for a good reason (maybe that's a wrong assumption).

8fb8f8916b122c7059af8ca2f4b646db26fb7272

Reply:
Except it'll still re-add the removed label when they push another commit to the PR, or edit the message, or add any other label...